processor/datadogprocessor/README.md

```diff
@@ -0,0 +1,27 @@
+# Datadog Processor
+
+| Status                   |               |
+|--------------------------|---------------|
+| Stability                | [beta] |
```

Comment:
This needs to be `in development` for now

Reply:
Sure. That's what I wanted to add initially, but felt it wasn't really accurate because according to the descriptions (which I can't find anymore). I changed it, but what's wrong with beta? This is working fine.

Reply:
Until this is added to the manifest on `opentelemetry-collector-releases` I don't think we should say this is beta, since it's not usable by users of the official builds.